.github/workflows/sycl_precommit.yml

```diff
@@ -28,3 +28,17 @@ jobs:
 
     - name: Check if clang-format patch is empty
       run: bash -c "if [ -s ./clang-format.patch ]; then cat ./clang-format.patch; exit 1; fi"
+
+  linux_default:
+    name: Default Linux checks
+    # Only build and test patches, that have passed all linter checks, because
+    # the next commit is likely to be a follow-up on that job.
```

Comment:
How about this:
1. `pulldown` label disables linters completely, because these checks always fail on community code, and we're not going to fix them anyway. In future we can add more logic to this label (like, running additional testing, or some fancy automation).
2. `ignore-lint` label does exactly that. The job fails, but the testing will be done anyway.

Reply:
"pulldown" usually contains our patches our team commits to adjust our code to community changes. These must be properly formatted. It might be difficult to separate these commits from the "community" changes, but if it's possible, running clang-format still has value.

Reply:
Ok, removed